tests/tests/service/v4/SugarWebServiceUtilv4Test.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace SuiteCRM;
+
+use aCase;
+use SugarWebServiceUtilv4_1;
+
+class SugarWebServiceUtilv4Test extends StateCheckerPHPUnitTestCaseAbstract
```

Comment:
please use `SuiteCRM\StateCheckerUnitAbstract` as parent class since we using codeception.